src/Dfe.Spi.Common/Dfe.Spi.Common.Caching/Definitions/Managers/IMemoryCacheManager{TCacheKey,TManagerItem}.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Dfe.Spi.Common.Caching.Definitions.Managers
+{
+    /// <summary>
+    /// Describes the operations of the memory cache manager.
+    /// </summary>
+    /// <typeparam name="TCacheKey">
+    /// The type of key used in the underlying storage.
+    /// </typeparam>
+    /// <typeparam name="TManagerItem">
+    /// The type of item being managed.
+    /// </typeparam>
+    public interface IMemoryCacheManager<TCacheKey, TManagerItem>
```

Comment:
Why do we need a Typed cache managers?

Reply:
Removed. Code updated to use strings and objects, as suggested.